stash/src/scraper/PoolsScraper.ts

```diff
@@ -9,13 +9,16 @@ import { ITuple } from '@polkadot/types-codec/types/interfaces'
 import { u32, u128 } from '@polkadot/types-codec/primitive'
 import { Option } from '@polkadot/types-codec/base'
 import { BN } from '@polkadot/util'
+import { toHuman } from '../util/Chain.js'
 
 let assets: Map<string, Asset> = new Map()
 
 export const fetchPools = async (block: Block): Promise<void> => {
   await initAssets(block)
   const pools = await getPools(block)
-  await store.savePools(pools)
+  if (pools !== null && pools !== undefined && pools.length > 0) {
```

Comment:
When we have no pools defined the app is crashing so this was introduced to prevent the crash.